dask/array/tests/test_linalg.py

```diff
@@ -760,6 +760,7 @@ def _get_symmat(size):
     return lA.dot(lA.T)
 
 
+@pytest.mark.xfail(reason="https://github.com/dask/dask/issues/9335", strict=False)
```

Comment:
Can we make it so this only is expected to fail when the appropriate version of `scipy` is installed? I think https://docs.pytest.org/en/7.1.x/how-to/skipping.html#condition-parameter is what we're after for that

Reply:
Thanks, yes!